### DISCUSSION TITLE
Hide footer in /docs

### DISCUSSION THREAD

**Discussion by chilikasha** [facebook/docusaurus#4557] [Q&A]
Hi, is it possible to hide footer from docs page?
sticky navbar is not very visible in some cases (like on screenshot).
![2021-04-02 19 21 22](https://user-images.githubusercontent.com/3147449/113434281-5654e780-93e9-11eb-86ac-5bac43640809.jpg)

on maybe modify navbar colors, so they are different from footer

**Comment by slorber** [comment]
Not sure to understand what you mean sorry.

You can add custom CSS to your Docusaurus site and change any color you want, or hide the foote with custom CSS too if you want.

**Reply by dev-maheshr** [reply]
what is the inbuilt tag for hiding the footer in a page, and where we need to change it
